### PR TITLE
SettingsPage

### PR DESCRIPTION
**Pull request by Madara0330E**
добавил пустую страницу настроек и путь для настроек в роутере

### CODE REVIEW

File: luckydiamond/src/assets/css/PagesStyles/SettingsPage.css
Comment (CkutlsGit) [inline luckydiamond/src/assets/css/PagesStyles/SettingsPage.css:1]: ##  **The file should not have any styles, it should be empty. These styles are useless and do not change the page styles in any way.**

File: luckydiamond/src/components/AsidebarComponent.vue
Comment (CkutlsGit) [inline luckydiamond/src/components/AsidebarComponent.vue:1]: ## **Not bad. But there was no point in deleting the "#" in the 5th line.**

File: luckydiamond/src/pages/SettingsPage.vue
Comment (CkutlsGit) [inline luckydiamond/src/pages/SettingsPage.vue:1]: ## **Bad semantics, a lot of useless code. The page should only have: AsideBar, ChatComponent, HeaderComponent and a section for Settings.**

File: luckydiamond/src/router/router.js
Comment (CkutlsGit) [inline luckydiamond/src/router/router.js:1]: **Not bad. But it's worth changing the style of the name for settingsPage, instead it should be PascalCase: "SettingsPage". The path should not contain "ssetting", but "settings". There should preferably be an indentation between the 7th and 8th line.**

Comment (danilt2000) [inline luckydiamond/src/router/router.js:1]: Typo '/ssetting' - > /settings

**Review by CkutlsGit — COMMENTED** [review]
## **It needs to be fixed:**
- Remove all styles in the SettingsPage.css file
- Change the template for the SettingsPage. What should be on it is written in the comment.
- Change the style in the router as written in the comment.
_@Madara0330E_

**Pull request closed without merging**